alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -56,10 +57,16 @@ Proceed with the following steps to fix the issue:
 1. Use the logs to guide you to the place where the conflicting targets are defined.
 2. Remove the duplicated targets or ensure that distinct targets are labeled uniquely.
 
-### The logs only revolve around the same scrape pool
+### Target is exposing duplicated samples for the same timestamp
+
+In this scenario, even if the samples have the same value for the same
+timestamp, they are considered duplicates.
```

Comment:
Actually, having duplicates for a timestamp with the same values would trigger the alert as well, as it may be hiding an issue on metrics exposition the target.
Of course, samples with explicit timestamps would not be concerned.